<testsuites>
	<testsuite id="1" tests="18" failures="6" package="D:\Relimetrics\Projects\RelimetricsGaugeApiProject\Api_Testing\specs\Relimetrics\UserSpecsV2\CreateUserTestV2.spec" time="19.540" timestamp="2023-02-06T00:28:00" name="check multiple user  fields" errors="0" hostname="DESKTOP-QLPIVJU">
		<properties></properties>
		<testcase classname="check multiple user  fields" name="create user with data above 1" time="3.826">
			<failure message="Check &#34;usersCountBefore&#34; and &#34;usersCountAfter&#34; depends on &lt;Expected value&gt;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;5&gt; but was: &lt;4&gt;&#39;" type="Check &#34;usersCountBefore&#34; and &#34;usersCountAfter&#34; depends on &lt;Expected value&gt;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;5&gt; but was: &lt;4&gt;&#39;">org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)&#xA;org.junit.jupiter.api.AssertionUtils.failNotEqual(AssertionUtils.java:62)&#xA;org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)&#xA;org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)&#xA;org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:527)&#xA;imp.JsonFilterImp.jsonArrayFilterScenarioUserCount(JsonFilterImp.java:100)&#xA;java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#xA;java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)&#xA;java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#xA;java.base/java.lang.reflect.Method.invoke(Method.java:568)&#xA;com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:28)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)&#xA;java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)&#xA;java.base/java.lang.Thread.run(Thread.java:833)&#xA;</failure>
		</testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 2" time="1.039"></testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 3" time="1.065">
			<failure message="Check &#34;usersCountBefore&#34; and &#34;usersCountAfter&#34; depends on &lt;Expected value&gt;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;4&gt; but was: &lt;5&gt;&#39;" type="Check &#34;usersCountBefore&#34; and &#34;usersCountAfter&#34; depends on &lt;Expected value&gt;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;4&gt; but was: &lt;5&gt;&#39;">org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)&#xA;org.junit.jupiter.api.AssertionUtils.failNotEqual(AssertionUtils.java:62)&#xA;org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)&#xA;org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)&#xA;org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:527)&#xA;imp.JsonFilterImp.jsonArrayFilterScenarioUserCount(JsonFilterImp.java:102)&#xA;java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#xA;java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)&#xA;java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#xA;java.base/java.lang.reflect.Method.invoke(Method.java:568)&#xA;com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:28)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)&#xA;java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)&#xA;java.base/java.lang.Thread.run(Thread.java:833)&#xA;</failure>
		</testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 4" time="1.017"></testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 5" time="0.936">
			<failure message="Get &#34;responseValue&#34; from scenario store and check if it is contains &lt;Expected value&gt; as string&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: username is already exists isn&#39;t contains key should not be empty as String ==&gt; expected: &lt;true&gt; but was: &lt;false&gt;&#39;" type="Get &#34;responseValue&#34; from scenario store and check if it is contains &lt;Expected value&gt; as string&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: username is already exists isn&#39;t contains key should not be empty as String ==&gt; expected: &lt;true&gt; but was: &lt;false&gt;&#39;">org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)&#xA;org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:40)&#xA;org.junit.jupiter.api.Assertions.assertTrue(Assertions.java:210)&#xA;imp.MutualCompareImp.dataCompareContainsDirectData(MutualCompareImp.java:128)&#xA;java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#xA;java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)&#xA;java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#xA;java.base/java.lang.reflect.Method.invoke(Method.java:568)&#xA;com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:28)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)&#xA;java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)&#xA;java.base/java.lang.Thread.run(Thread.java:833)&#xA;</failure>
		</testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 6" time="1.011">
			<failure message="Check &#34;usersCountBefore&#34; and &#34;usersCountAfter&#34; depends on &lt;Expected value&gt;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;4&gt; but was: &lt;5&gt;&#39;" type="Check &#34;usersCountBefore&#34; and &#34;usersCountAfter&#34; depends on &lt;Expected value&gt;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;4&gt; but was: &lt;5&gt;&#39;">org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)&#xA;org.junit.jupiter.api.AssertionUtils.failNotEqual(AssertionUtils.java:62)&#xA;org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)&#xA;org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)&#xA;org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:527)&#xA;imp.JsonFilterImp.jsonArrayFilterScenarioUserCount(JsonFilterImp.java:102)&#xA;java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#xA;java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)&#xA;java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#xA;java.base/java.lang.reflect.Method.invoke(Method.java:568)&#xA;com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:28)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)&#xA;java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)&#xA;java.base/java.lang.Thread.run(Thread.java:833)&#xA;</failure>
		</testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 7" time="0.947">
			<failure message="Check &#34;usersCountBefore&#34; and &#34;usersCountAfter&#34; depends on &lt;Expected value&gt;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;4&gt; but was: &lt;5&gt;&#39;" type="Check &#34;usersCountBefore&#34; and &#34;usersCountAfter&#34; depends on &lt;Expected value&gt;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;4&gt; but was: &lt;5&gt;&#39;">org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)&#xA;org.junit.jupiter.api.AssertionUtils.failNotEqual(AssertionUtils.java:62)&#xA;org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)&#xA;org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)&#xA;org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:527)&#xA;imp.JsonFilterImp.jsonArrayFilterScenarioUserCount(JsonFilterImp.java:102)&#xA;java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#xA;java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)&#xA;java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#xA;java.base/java.lang.reflect.Method.invoke(Method.java:568)&#xA;com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:28)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)&#xA;java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)&#xA;java.base/java.lang.Thread.run(Thread.java:833)&#xA;</failure>
		</testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 8" time="0.927"></testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 9" time="0.905"></testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 10" time="0.888"></testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 11" time="0.907"></testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 12" time="0.894"></testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 13" time="0.910"></testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 14" time="0.934"></testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 15" time="0.892"></testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 16" time="0.920"></testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 17" time="0.618">
			<failure message="Check if status code is &lt;Status code&gt;&#xA;Step Execution Failure: &#39;java.lang.AssertionError: 1 expectation failed.&#xA;Expected status code &lt;200&gt; but was &lt;422&gt;.&#xA;&#39;" type="Check if status code is &lt;Status code&gt;&#xA;Step Execution Failure: &#39;java.lang.AssertionError: 1 expectation failed.&#xA;Expected status code &lt;200&gt; but was &lt;422&gt;.&#xA;&#39;">java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#xA;java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)&#xA;java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#xA;java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)&#xA;java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)&#xA;org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)&#xA;org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)&#xA;org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)&#xA;org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)&#xA;org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)&#xA;io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:493)&#xA;io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)&#xA;io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:674)&#xA;jdk.internal.reflect.GeneratedMethodAccessor156.invoke(Unknown Source)&#xA;java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#xA;java.base/java.lang.reflect.Method.invoke(Method.java:568)&#xA;org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)&#xA;org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)&#xA;org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)&#xA;org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)&#xA;io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:126)&#xA;io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)&#xA;io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:134)&#xA;io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)&#xA;io.restassured.internal.ValidatableResponseImpl.super$2$statusCode(ValidatableResponseImpl.groovy)&#xA;jdk.internal.reflect.GeneratedMethodAccessor155.invoke(Unknown Source)&#xA;java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#xA;java.base/java.lang.reflect.Method.invoke(Method.java:568)&#xA;org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)&#xA;groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)&#xA;groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1267)&#xA;org.codehaus.groovy.runtime.ScriptBytecodeAdapter.invokeMethodOnSuperN(ScriptBytecodeAdapter.java:144)&#xA;io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy:142)&#xA;io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy)&#xA;helper.StatusCodeHelper.checkStatusCode(StatusCodeHelper.java:26)&#xA;imp.StatusCodeImp.checkStatusCodeFromRes(StatusCodeImp.java:16)&#xA;jdk.internal.reflect.GeneratedMethodAccessor154.invoke(Unknown Source)&#xA;java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#xA;java.base/java.lang.reflect.Method.invoke(Method.java:568)&#xA;com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:28)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)&#xA;java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)&#xA;java.base/java.lang.Thread.run(Thread.java:833)&#xA;</failure>
		</testcase>
		<testcase classname="check multiple user  fields" name="create user with data above 18" time="0.904"></testcase>
		<system-out></system-out>
		<system-err></system-err>
	</testsuite>
</testsuites>